spec/api/comment_thread_spec.rb

```diff
@@ -258,22 +258,58 @@ def move_to_front(ary, *vals)
             expected_order = @default_order
             actual_order.should == expected_order
           end
-          it "sorts using last activity / descending" do
+          it "sorts using last activity / descending when thread is updated" do
+            t5 = @threads["t5"]
+            t5.update(body: "changed!")
+            t5.save!
+            actual_order = thread_result_order("activity", "desc")
+            expected_order = @default_order
+            actual_order.should == expected_order
+          end
+          it "sorts using last activity / ascending when thread is updated" do
+            t5 = @threads["t5"]
+            t5.update(body: "changed!")
+            t5.save!
+            actual_order = thread_result_order("activity", "asc")
+            expected_order = @default_order.reverse
+            actual_order.should == expected_order
+          end
+          it "sorts using last activity / descending when comment is updated" do
             t5c = @threads["t5"].comments.first
             t5c.update(body: "changed!")
             t5c.save!
             actual_order = thread_result_order("activity", "desc")
             expected_order = @default_order
             actual_order.should == expected_order
           end
-          it "sorts using last activity / ascending" do
+          it "sorts using last activity / ascending when comment is updated" do
```

Comment:
FYI: Ascending sort will probably be removed soon.  Nothing to do.  Just letting you know.